components/MeetingDetail/MeetingDetail.js

```diff
@@ -0,0 +1,61 @@
+import styled from "styled-components";
+import DogName from "../DogName/DogName";
+import ImageWithText from "../ImageWithText/ImageWithText";
+
+export default function MeetingDetail({ data }) {
+  const { location, date, time, excluded, furtherInfo, attending } = data;
+  return (
+    <StyledArticle>
+      <h2>{location}</h2>
+      <ImageWithText
+        image="calendar-icon.svg"
+        altText="Kalender Icon"
+        text={date}
+      />
+      <ImageWithText image="clock-icon.svg" altText="Uhr Icon" text={time} />
+      {excluded ? (
+        <ImageWithText
+          data-testid="excluded"
+          image="stop-icon.svg"
+          altText="Stop Icon"
+          text={excluded}
+        />
+      ) : null}
+      <StyledInfobox>
+        {furtherInfo ? (
+          <div data-testid="info">
+            <label htmlFor="weitereInfos">Weitere Infos:</label>
+            <StyledParagraph id="weitereInfos">{furtherInfo}</StyledParagraph>
+          </div>
+        ) : null}
```

Comment:
Instead of the ternary, you could use the logical `&&` operator:
```suggestion
      {excluded && (
        <ImageWithText
          data-testid="excluded"
          image="stop-icon.svg"
          altText="Stop Icon"
          text={excluded}
        />
      )}
      <StyledInfobox>
        {furtherInfo && (
          <div data-testid="info">
            <label htmlFor="weitereInfos">Weitere Infos:</label>
            <StyledParagraph id="weitereInfos">{furtherInfo}</StyledParagraph>
          </div>
        )}
```

Reply:
Thanks for this adivce! I adapted my code.

Reply:
You can resolve these conversations by clicking the "Resolve conversation" button below; by doing so, you will get a better overview what's happening in your PR.